Jenkinsfile

```diff
@@ -729,6 +840,28 @@ pipeline {
             }
             parallel {
                 stage('Unit Test on EL 8') {
+                    when {
+                        beforeAgent true
+                        expression { !skipStage() }
+                    }
+                    agent {
+                        label cachedCommitPragma(pragma: 'VM1-label', def_val: params.CI_UNIT_VM1_LABEL)
+                    }
+                    steps {
+                        job_step_update(
+                            unitTest(timeout_time: 60,
+                                     unstash_opt: true,
+                                     inst_repos: prRepos(),
+                                     inst_rpms: unitPackages()))
+                    }
+                    post {
+                        always {
+                            unitTestPostEx(['unit_test_logs/'])
+                            job_status_update()
```

Comment:
A pipeline-lib routine can not call job_status_update due to variable scope.
The unitTestPostEx should be able to call it, which would simplify the stages.

Reply:
I made this change to test.  We will need to add a ccov stage at some point for the bdev tests so every little bit can help.